pom.xml

```diff
@@ -124,6 +124,16 @@
 			<version>3.2.5</version>
 		</dependency>
 		<dependency>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>commons-vfs2</artifactId>
+			<version>2.0</version>
```

Comment:
I see there also is a 2.2 version, are you sticking to 2.0 intentionally?

Reply:
No, when we added the dependency locally, the 2.0 was the latest version.